src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {

```

Comment:
There is an empty line at the beginning of the class implementation, please remove it.